content/blog/2019-02-06-interview-with-hostinger.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Interview with Hostinger
+created_at: 2019-02-06
+kind: article
+author_name: Brian Brazil
+---
+
+*Continuing our series of interviews with users of Prometheus, Donatas Abraitis
+from Hostinger talks about their monitoring journey.*
+
+## Can you tell us about yourself and what Hostinger does?
+
+I’m Donatas Abraitis, a systems engineer at
+[Hostinger](https://www.hostinger.com/). Hostinger is the hosting company as
+the name implies. We have around 30M clients since 2004 including
+the [000webhost.com](https://www.000webhost.com/) project - free web hosting provider.
+
+## What was your pre-Prometheus monitoring experience?
+
+When Hostinger was quite a small company, only Nagios, Cacti and Ganglia
+existed at that time in the market as open source monitoring tools. This is
+like telling young people what a floppy drive is, but Nagios and Cacti are
+still in development cycle today.
+
+
+Even though no automation tools existed. Bash + Perl did the job. If you want
+to scale your team and yourself, automation should never be ignored. No
+automation - more human manual work involved.
+
+At that time there were around 150 physical servers. To compare, till this day
+we have around 2000 servers including VMs and physical boxes.
+
+For networking gear, SNMP is still usable worldwide, but currently, with
```

Comment:
Run-on sentence.

How about:  `For networking gear, SNMP is still widely used. With the rise of "white box" switches SNMP becomes less necessary, as regular tools can be installed."